src/assets/templates/widgets/finaid_awards_section.html

```diff
@@ -81,27 +81,33 @@ <h3 data-ng-bind="section.total.title" class="cc-page-finaid-header"></h3>
           </div>
           <div data-ng-if="item.subItems.remainingAmount !== undefined" class="cc-widget-finaid-awards-section-item-margin-top">
             <strong>Remaining Work-Study:</strong>
-            <span data-cc-amount-directive="item.subItems.remainingAmount" data-cc-amount-directive-color="false" data-cc-amount-directive-initial-space="false"></span>
+            <span data-cc-amount-directive="finaidAwardsInfo.formatCurrency(item.subItems.remainingAmount)" data-cc-amount-directive-color="false" data-cc-amount-directive-initial-space="false"></span>
           </div>
           <div data-ng-if="item.subItems.termDetails.length" class="cc-widget-finaid-awards-section-item-margin-top">
             <div class="cc-table">
               <table>
                 <thead>
                   <tr>
-                    <th scope="col">Term</th>
+                    <th scope="col">
+                      <span data-ng-bind="finaidAwardsInfo.checkForDisbursementDates(item.subItems.termDetails)"></span>
+                    </th>
                     <th scope="col" class="cc-table-right">Offered</th>
                     <th scope="col" class="cc-table-right">Disbursed</th>
                   </tr>
                 </thead>
                 <tbody>
                   <tr data-ng-repeat="termItem in item.subItems.termDetails">
-                    <td data-ng-bind="termItem.term"></td>
-                    <td class="cc-table-right" data-cc-amount-directive="termItem.offered" data-cc-amount-directive-color="false"></td>
-                    <td class="cc-table-right" data-cc-amount-directive="termItem.disbursed" data-cc-amount-directive-color="false"></td>
+                    <td data-ng-if="!termItem.disbursementDate" data-ng-bind="termItem.term"></td>
+                    <td data-ng-if="termItem.disbursementDate" data-ng-bind="termItem.disbursementDate"></td>
+                    <td class="cc-table-right" data-cc-amount-directive="finaidAwardsInfo.formatCurrency(termItem.offered)" data-cc-amount-directive-color="false"></td>
+                    <td class="cc-table-right" data-cc-amount-directive="finaidAwardsInfo.formatCurrency(termItem.disbursed)" data-cc-amount-directive-color="false"></td>
                   </tr>
                 </tbody>
               </table>
             </div>
+            <p data-ng-show="finaidAwardsInfo.checkForDisbursementDates(item.subItems.termDetails) === 'Est. Disbursement'"
+              data-ng-bind-html="finaidAwards.messages.messageEstDisbursements"
+              style="margin-top:5px"></p>
```

Comment:
@marcpechaitis - Inline style?

Reply:
@redconfetti is that one-off style better than a generic `cc-margin-top-5px`?

cc: @developish because I did the inline during consultation with him

Reply:
Discussed with @marcpechaitis - Letting this go in as-is. We do need to have a discussion about our front-end standards, in respect to CSS, as we migrate to React components. cc: @developish